README.md

```diff
@@ -13,7 +13,7 @@ composer require symfony/webpack-encore-bundle
 
 ## Configuration
 
-If you're using Symfony Flex, you're done! The recipe will
+If you're using Symfony Flex, you're done! If not, [enable the bundle](https://symfony.com/doc/current/bundles.html) as normal. The recipe will
```

Comment:
I'd add this line after the code block of the config example, and also add something like "... and copy/paste the config example from above"

Before, those 2 sentences were too related to each other, and now the new sentence just breaks them. Any thoughts?